### DISCUSSION TITLE
unloaded tabs are loading on tab drag when merely hovering

### DISCUSSION THREAD

**Discussion by Elementari** [mbnuqw/sidebery#1237] [General]
Hello,

I was dragging a tab group across many dozens of tabs, and whilst I was dragging, many tabs I was passing started to load (they had all been unloaded (gray)). This caused the system to slow down and also basically made to restart the browser because I did not want all those tabs loaded in memory. 

Why does this happen? Why not leave those tabs unloaded?

**Comment by JohnVKRDam** [comment]
Hi there,

Isn't that because of the Drag & Drop Settings? Specifically the **_Activate Tab on Hover_** settings?
I had similar issues with tabs loading while I was dragging stuff, and disabled the Hover setting entirely.
But, if you want to keep that setting enabled, you could set it to a higher delay (or the **_With Pressed Key_**).

I particularly had issues with folded branches unfolding while dragging tabs over them, so I set the **_Expand/Fold the Branch on Hovering over the ..._** to a delay of 1500ms. That greatly helped in preventing the unintentional unfolding of branches  :)

See print-screen below, of the Sidebery / Settings.

I hope that helps 👍 
John

![image](https://github.com/mbnuqw/sidebery/assets/43664932/3dbd9cba-813d-49bb-b7da-4fd3fdb54fa9)
